.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    root: true,
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module'
+    },
+    extends: 'eslint:recommended',
```

Comment:
Also, I never thought I'd see the day you turned on a style checker that was ok with var. ;)

Ah well, presets are what they are.